.github/policies/resourceManagement.yml

```
@@ -152,5 +152,18 @@ configuration:
       - addReply:
           reply: Hello! I noticed that you're targeting one of our servicing branches. Please consider updating the version.
       description: Comment on vs* branches
+    - if:
+      - payloadType: Pull_Request
+      - or:
+        - isAction:
+            action: Opened
+        - isAction:
+            action: Reopened
+      - filesMatchPattern:
+          pattern: ^.+\.swr$
+      then:
+      - addReply:
+          reply: Hello @${issueAuthor}, I noticed that you’re changing *.swr file. Please make sure to run VS Perf DDRITs using exp/* insertion before merging the change.

```

I wonder if a slightly longer comment would make it clearer, especially to folks who are not familiar with our processes:

```suggestion
          reply: Hello @${issueAuthor}, I noticed that you’re changing an *.swr file. Please make sure to validate this change by an experimental VS insertion. This is accomplished by pushing to an exp/* branch, which requires write permissions to this repo.
```